datatransferobjects/src/main/java/PassengerDTO.java

```diff
@@ -0,0 +1,39 @@
+import java.sql.Date;
+import java.util.Collection;
+
+public class PassengerDTO {
```

Comment:
Is it DTO object or it's SO object?
If it DTO please implement this object with immutable properties